packages/cozy-stack-client/src/CozyStackClient.js

```diff
@@ -21,7 +22,10 @@ const normalizeUri = uri => {
  * Main API against the `cozy-stack` server.
  */
 class CozyStackClient {
-  constructor({ token, uri = '' }) {
+  constructor(options) {
+    const opts = { ...options }
```

Comment:
What is the purpose if this new object instead of using `options` directly?

Reply:
Since we write in options (in CozyClient :/), I prefer to make a copy.